documentation/release-notes/topics/version_5.md

```diff
@@ -14,46 +14,75 @@ This section covers significant changes and bug fixes in Hyperview 5.1.x since v
 
-Manual environmental sensor (multiple sensors per device)
+Please refer to the Hyperview product documentation and user guide for more information.
+
+### Enhanced Feature: Asset Dashboard
+
+This new widget (Associations) displays asset association to business entities, linked sensors, power providers and network connections and circuits. It provides one view to see these dependencies and connections in one place.
 
 ### Enhanced Feature: Catalog
 
-Model cloning in catalog
+This features improves the usability of the catalog by allowing users that want to customize existing models to easily clone existing data and perform the needed customization, for example, chance product images.
 
-### Enhanced Feature: Modular Patch Panels
+### Enhanced Feature: Chart time range selector
 
-### Enhanced Feature: Chart Time Range Selector
+The time range selector for charts has been improved to add support for local time based charts where possible. Previously, The time range selector allowed only for UTC based time ranges.
 
-### Enhanced Feature: 2D and 3D Layout
+### Enhanced Feature: Data Grids
+
+Grid double click action has been improved to be more consistent and available on more more data grids. This will improve the navigation experience within the product.
+
+Grids now have 2 modes, an adaptive mode, which is the existing default, where rows would compress to should applicable data depending on the width of the screen. And a non-adaptive mode, where grids will enable horizontal scrolling to show columns extra column. The mode is activated by a button next to the grid column selector.
 
-Custom properties filter in layout
+### Enhanced Feature: DC Power / Rectifier Support
 
-### Enhanced Feature: DC Energy Management
+A new asset type has been added for DC rectifiers for DC-powered data centers.
 
-Rectifier support for telco deployments
+The discovery engine has been enhanced to add support for DC rectifier discovery and monitoring.
+
+The power analysis engine for location has also been improved to take DC power in various power analysis pipelines if configured to do so.
 
 ### Enhanced Feature: Localization
 
-French language support
+French language support has been added to the product UI.
+
+### Enhanced Feature: Manual Sensors
+
+Customers that are managing the telemetry from temperature and humidity sensors, can now add more than one sensor per device. Prior to this release assets were limited to one manual temperature and humidity sensor.
+
+### Enhanced Feature: Patch Panel Management
+
+Support for patch panels has been enhanced. Modular patch panels can now have extra meta data that would make it easier to manage and report on slot number, module meta data and other useful attributes.
```

Comment:
The term "meta data" should be "metadata".

Reply:
Fixed, thank you!